packages/lib/file-api-driver-webdav.js

```diff
@@ -224,6 +226,13 @@ class FileApiDriverWebDav {
 		await this.delete('');
 		await this.mkdir('');
 	}
+
+	initialize() {
+		const providerCheck = checkProviderIsUnsupported(Setting.value('sync.6.path'));
+		if (providerCheck.isUnsupported === true) {
```

Comment:
Just `if (providerCheck.isUnsupported)` when checking booleans or null values (I will add an eslint rule for this)

Reply:
Fixed.